uint32 id
sensor_msgs/NavSatFix[] waypoints
---
bool success
string message
